metals/src/main/scala/scala/meta/internal/rename/RenameProvider.scala

```diff
@@ -176,10 +176,11 @@ final class RenameProvider(
   ): Option[LSPEither[TextDocumentEdit, ResourceOperation]] = {
     fileChanges
       .find { file =>
-        isOccurence(str => {
+        isOccurence { str =>
+          str.owner.isPackage &&
```

Comment:
what happens if the file has no package definition?

Reply:
I believe it returns “empty/“

Reply:
I can see it's `_empty_/` :)